.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,51 @@
+name: presubmit
+on: [push, pull_request]
```

Comment:
Can you explain the why presubmit.yml and wheel.yml are separate?

Reply:
`wheel.yml` is generates a binary install package. I could put them together, but this allows changing them independently.

Reply:
Can you add a clear comment on what goes where?  The split isn't clear to me, and so I think we need some guidance on what goes where for future development?

Reply:
Done.